src/services/socket/api.ts

```diff
@@ -299,19 +339,16 @@ export class SocketAPI {
       }
     }
 
-    let response = await this.#fetch(`${this.#baseUrl}/route/start`, {
-      // @ts-ignore
-      method: 'POST',
-      headers: this.#headers,
-      body: JSON.stringify(params)
+    const response = await this.#handleResponse<SocketAPISendTransactionRequest>({
+      fetchPromise: this.#fetch(`${this.#baseUrl}/route/start`, {
+        method: 'POST',
+        headers: this.#headers,
+        body: JSON.stringify(params)
+      }),
+      errorPrefix: 'Unable to start the route.'
```

Comment:
Could we improve this message?

Reply:
The title of the card is "Pending route", the call to action is "Proceed", that's why "Unable to start the route" makes sense to me. I can sync up for all these messages at batch with Stefan down the road, closing this one now so as not to block the PR.